conf/test_virus_identification.config

```diff
@@ -0,0 +1,42 @@
+/*
+========================================================================================
+    Nextflow config file for running minimal tests
+========================================================================================
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/mag -profile test_virus_identification,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile for running virus_identification'
+    config_profile_description = 'Minimal test dataset to check pipeline function virus identification'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    input                       = 'https://raw.githubusercontent.com/nf-core/test-datasets/mag/samplesheets/samplesheet.csv'
```

Comment:
It depends what the purpose of the test is, but would it be valuable to test this with samples that include some viral sequences? AFAIK the mag test samples are simulated from some bacteria (??) so the test may return nothing - worth checking!

Reply:
There's some SARS-COV-2 fastq files in the modules/ test-data repo that might be easily re-purposed here? https://github.com/nf-core/test-datasets/tree/modules/data/genomics/sarscov2/illumina/fastq

Reply:
Hi @prototaxites thank you for the suggestion! I just looked at the test with the current dataset and it looks like there are some positive hits (most of these are an artifact of the tool's inability to accurately classify very short (<100bp) sequences, but also likely from some prophage artifacts within these bacterial genomes). Do you still think it would be best to change the test data?

Reply:
If it's passing some sequences, I'd say it's probably fine - as long as those can pass through CheckV without error once that's implemented!